Sources/SwiftDocC/SwiftDocCMarkdownOutput/MarkdownOutputManifest.swift

```diff
@@ -0,0 +1,98 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024-2025 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+// Consumers of `MarkdownOutputManifest` in other packages should be able to lift this file and be able to use it standalone, without any dependencies from SwiftDocC.
+
+/// A manifest of markdown-generated documentation from a single catalog
+@_spi(MarkdownOutput)
+public struct MarkdownOutputManifest: Codable, Sendable {
+    public static let version = "0.1.0"
+    
+    /// The version of this manifest
+    public let manifestVersion: String
```

Comment:
Like elsewhere, please use `SemanticVersion` rather than strings to represent structured data.